### ISSUE TITLE
[BF16] For Hugging Face benchmark model on stock pytorch, Softmax cost time on pvc-1100 worse than A100 * ratio

### ISSUE THREAD

**Issue by xiaowangintel** [intel/torch-xpu-ops#794]
### 🐛 Describe the bug

For more details, please refer to https://jira.devtools.intel.com/browse/PYTORCHDGQ-5010?filter=-2.

For more details, please refer to https://jira.devtools.intel.com/browse/PYTORCHDGQ-5018?filter=-2.

For more details, please refer to https://jira.devtools.intel.com/browse/PYTORCHDGQ-5092.
### Versions

pytorch commit:03480213dea1f60f6d12e7348904d2f3ef7314d0
torch-xpu-ops commit:718bc42c667539977e5eadb11ea4dec602544bf2
driver : hotfix_agama-ci-devel-881.19
pti : l_intel-pti-dev_p_0.9.0.38_offline.sh
basekit : l_BaseKit_p_2024.2.1.100_offline.sh

**Comment by retonym** [comment]
xpu performance is not targeted to PT 2.6

**Comment by weishi-deng** [comment]
Update the latest test for softmax on these models:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/wdeng/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/wdeng/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">


unit:ms | A100 | pvc 1100 | ratio
-- | -- | -- | --
T5Small | 18.09 | 32.781 | 55%
GPT2ForSequenceClassification | 18.152 | 34.974 | 52%
ElectraForQuestionAnswering | 13.466 | 20.277 | 66%



</body>

</html>

**Comment by Stonepia** [comment]
Target 2.8

**Comment by jianyizh** [comment]
I think after we enable sdpa, we won't meet these softmax

**Comment by majing921201** [comment]
Re-measured softmax performance, all meet the perf goal now. Previous low perf data should caused by profiling issue
1, [4, 12, 1024, 1024]
-------------------------------------------------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------
                                                   Name    Self CPU %      Self CPU   CPU total %     CPU total  CPU time avg      Self XPU    Self XPU %     XPU total  XPU time avg    # of Calls
-------------------------------------------------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------
                                         aten::_softmax        81.64%       4.780ms        99.42%       5.821ms       1.164ms       1.099ms       100.00%       1.099ms     219.808us             5
at::native::xpu::impl::DispatchSoftmaxForwardKernelF...         0.00%       0.000us         0.00%       0.000us       0.000us       1.099ms       100.00%       1.099ms     219.808us             5
                                          aten::softmax         0.58%      33.827us       100.00%       5.855ms       1.171ms       0.000us         0.00%       1.099ms     219.808us             5
                                       urUSMDeviceAlloc         3.06%     179.422us         3.06%     179.422us      89.711us       0.000us         0.00%       0.000us       0.000us             2
                                  urEnqueueKernelLaunch        14.72%     861.861us        14.72%     861.861us     172.372us       0.000us         0.00%       0.000us       0.000us             5
-------------------------------------------------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------
Self CPU time total: 5.855ms
Self XPU time total: 1.099ms

2, [64, 4, 512, 512]
-------------------------------------------------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------
                                                   Name    Self CPU %      Self CPU   CPU total %     CPU total  CPU time avg      Self XPU    Self XPU %     XPU total  XPU time avg    # of Calls
-------------------------------------------------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------
                                         aten::_softmax        81.94%       4.842ms        99.39%       5.874ms       1.175ms       1.434ms       100.00%       1.434ms     286.752us             5
at::native::xpu::impl::DispatchSoftmaxForwardKernelF...         0.00%       0.000us         0.00%       0.000us       0.000us       1.434ms       100.00%       1.434ms     286.752us             5
                                          aten::softmax         0.61%      35.898us       100.00%       5.909ms       1.182ms       0.000us         0.00%       1.434ms     286.752us             5
                                       urUSMDeviceAlloc         3.00%     177.202us         3.00%     177.202us      88.601us       0.000us         0.00%       0.000us       0.000us             2
                                  urEnqueueKernelLaunch        14.46%     854.390us        14.46%     854.390us     170.878us       0.000us         0.00%       0.000us       0.000us             5
-------------------------------------------------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------  ------------
